nautobot/docs/user-guides/getting-started/interfaces.md

```diff
@@ -0,0 +1,96 @@
+# Interfaces
+
+Interfaces can be added at the **Device** or the **Device Type** level.  Which one you choose depends on your use case.
+
+Interfaces added to an individual Device are for that Device only.
+Interfaces added to the Device Type will be applied to all NEW implementations of that Device Type (not existing implementations).  
+
+Which one you select depends on your use case; in some instances you will need to use both, as in the example below.
+
+## Interface Add Example
+
+Let’s take an example:
+
+* We want to define a Device Type of `MX240-edge`
+* This Device Type will have 20x10G (`xe[0-1]/0/[0-9]`) ports and one LAG (`ae0`)
+* The following ports will be part of `ae0`: `xe-0/0/9`, `xe-1/0/9`
+
+## Creating a Device Type
+
+We are going to use the **Device Type** to achieve part of this goal. Using the **Device Type** will also provide repeatability
+because the **Device Type** object also serves as a template. This templating feature is demonstrated in this example.
+
+Device Types can serve as templates for Devices, and as such the two are very similar. Here is a screenshot of a Device Type:
+
+![](../images/getting-started-nautobot-ui/21-device-type.png)
+
+Creating a Device Type is very similar to [creating a Device](creating-devices.md).
+
+To create a Device Type:
+
+1. Click on **Devices** in the top navigation menu
+2. Click on the **Device Types** option in the drop-down menu
+3. On the Device Types page `Add +` a new Device Type
+    * A Device Type requires a **Manufacturer** object to be created prior to creating the Device Type
+    * Device Type requires **Manufacturer**, **Model**, **Slug**, and **Height** values at creation.
+    * In this example, name the Device Type `MX240-edge`
+4. On the home page for the specific Device Type, click on `+Add Components` and select `Interfaces`    
+
+![](../images/getting-started-nautobot-ui/15-create-device-type.png)
+
+You will now see the `Interface Template` form:
+
+1. Add the `ae0` Interface Template
+    * `Manufacturer` will auto-populate to the Manufacturer of the Device Type you are editing
+    * `Device Type` will auto-populate to the Device Type you are editing
+    * Populate a `Name`
```

Comment:
Some inconsistency amongst the various docs about styling of references to form fields as "Name" versus "`Name`" versus "**Name**" - please pick one. :-)

Reply:
There are a couple that you caught.  I'll look at the other docs to make sure I'm consistent.

In general, the `` will refer to the field in the form. For example, fill in the `Manufacturer` on the form to be `Juniper`.  

Whereas "Manufacturer" is a generic reference to the Manufacturer object: The Manufacturer of the Device Type.

Bolding is used for menu navigation typically or to make things more evident in instructions.

`text` will also be used for buttons